### DISCUSSION TITLE
If possible, please support unocss

### DISCUSSION THREAD

**Discussion by miaojinsing** [saadeghi/daisyui#3029] [Ideas / feature requests]
Unocss can better integrate with modern development tools. If possible, we hope you can support it.

**Comment by saadeghi** [comment]
UnoCSS is not compatible with Tailwind CSS plugins.  
It requires something called `preset` which is totally different from Tailwind's `plugin` API. providing a daisyUI preset for UnoCSS would basically need a copy of all styles with a different syntax and at the end we will lose a lot of functionalities Tailwind CSS API is providing.

A daisyUI preset for UnoCSS would be a totally different project. There are some presets for daisyUI made by the community:
- https://github.com/kidonng/unocss-preset-daisy
- https://github.com/unscatty/unocss-preset-daisy
- https://github.com/Super-Lollipop/unocss-preset-daisyui

The first one has a PR for daisyUI 4 update: https://github.com/kidonng/unocss-preset-daisy/pull/39, you can probably fork it and use it in your project.

Let me know if you have a question.
